cwltool/utils.py

```diff
@@ -19,9 +24,93 @@ def get_feature(self, feature):  # type: (Any, Any) -> Tuple[Any, bool]
             return (t, True)
     for t in reversed(self.hints):
         if t["class"] == feature:
+            if "requirement" in t and t["requirement"] is True:
+                return(t, True)
```

Comment:
This is a big behavior change. I'm guessing that this is about enforcing priority of the dockerRequirements, but now this method works rather differently than before. Perhaps a rename is called for?